study.md

```diff
@@ -66,7 +68,7 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+.where
```

Comment:
`#where` will work without any filters passed in, but `#all` will always return every record in a collection.